python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -408,7 +457,10 @@ async def call_pipetask(self, image_type, exposure_ids_dict):
              Dictionary with the final OCPS status.
         """
 
-        # Run the pipetask via the OCPS
+        # Run the pipetasks via the OCPS.
+        # By default, config.generate_calibrations is 'false'
+        # and the necessary calibrations are assumed to be
+        # in the input collections.
 
         if image_type == "BIAS":
```

Comment:
Thanks for the suggestion.  I'll have to include the input collections and options for each type as parameters of the function since they are different in each {{config_string}}, but that still should be better than how it is right now.  I assume that you would suggest doing something similar in the function {{verify_calib}}, correct?